### DISCUSSION TITLE
SMplayer spans screens when using multiple monitors

### DISCUSSION THREAD

**Discussion by harshitgarg22** [smplayer-dev/smplayer#101] [Q&A]
When I'm using SMplayer on a dual monitor setup, the fullscreen videos spans across the monitors. Is there a way to stop this from happening? I have searched across the internet and settings and to the best of my knowledge there is no solution (even barely a mention) of this.

![Screenshot_20210523_133757](https://user-images.githubusercontent.com/16517449/119252863-1d0c4d00-bbcc-11eb-9eb1-7efc7434bb43.png)
Doesn't look very aesthetically pleasing. (Its across two screens so its not obvious how erratic it looks)

**Comment by smplayer-dev** [comment]
What operating system are you using?
I never experienced this problem.

**Comment by harshitgarg22** [comment]
I'm using Manjaro+KDE. Is there a setting I perhaps might have changed inadvertently? I've gone through and can't seem to find any.

**Reply by smplayer-dev** [reply]
I don't know. Do other applications also span multiple screens in fullscreen mode?

**Reply by harshitgarg22** [reply]
No something like firefox doesn't do this. Is there any log or anything I can provide to help with this? Also would probably be helpful if someone else with Manjaro+KDE can report what's happening with them.

**Reply by smplayer-dev** [reply]
I've just tested on Manjaro KDE in a virtual machine, with 2 screens. Fullscreen only use one screen. So I don't know, it must be something in your configuration.

**Reply by harshitgarg22** [reply]
Argh I just found the problem. In KWin Scripts there's a setting "Video Wall" which is supposed to span video across two screens. Found solution here https://bbs.archlinux.org/viewtopic.php?id=169628

Thank you so much for your timely replies.
